### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#788] [Releases]
### Release v0.12.0-beta.1

* [Feature] PHP 8.2 and performance improvements by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/676
* Release v0.12.0-beta.1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/729

### Release v0.12.0-beta.2

* [Bugfix] Docker image builds by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/739
* Release v0.12.0-beta.2 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/740

### Release v0.12.0-beta.3

* [Bugfix] Missing "v" from build versions by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/743
* Release v0.12.0-beta.3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/744

### Release v0.12.0-beta.4

* [Feature] Generate app key on startup by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/746
* [Feature] Filament v3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/736
* Release v0.12.0-beta.4 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/757

### Release v0.12.0-beta.5

* [Chore] Hint links by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/759
* Release v0.12.0-beta.5 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/760

### Release v0.12.0-beta.6
 
* [Feature] User role by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/762
* Laravel 10.23.0 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/764
* Laravel 10.23.1 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/768
* Laravel 10 Mid-Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/771
* added back collapsible sidebar by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/772
* [Bug] Settings page authorization by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/773
* Release v0.12.0.beta.6 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/774

### Release v0.12.0

* [Bug] Removed guest role by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/776
* [Bugfix] Mount parent method on settings pages by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/785
* Release v0.12.0 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/787

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.11.22...v0.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by GiuffreLab** [comment]
It appears the upgrade to v0.12.0 has removed all of the settings options for those who removed the admin default user and created their own in previous versions. Was this an oversite or has the access to settings been hidden somewhere that I'm just blind and not seeing>

**Reply by alexjustesen** [reply]
Check out the CLI section in the docs. This will help resolve the issue and correct access. 

I'm assigning the first user as the admin as of this release and if you're not using the first user then access wouldn't have been provisioned for your use case. 

Impossible to account for everyone unfortunately.
